paws_collector.js

```diff
@@ -378,7 +378,17 @@ class PawsCollector extends AlAwsCollector {
                 return collector.checkStateSqsMessage(stateSqsMsg, asyncCallback);
             },
             function(asyncCallback) {
-                return collector.pawsGetLogs(pawsState.priv_collector_state, asyncCallback);
+                return collector.pawsGetLogs(pawsState.priv_collector_state, (err, ...remainingParams) => {
```

Comment:
Should probably add tests for this new flow

Reply:
For now I have added the test cases for newly added methods .I will check if i can add more test cases to test flow after my vacation.